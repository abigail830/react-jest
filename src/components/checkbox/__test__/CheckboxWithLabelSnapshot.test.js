// Link.react.test.js
import React from 'react';
import CheckboxWithLabel from '../CheckboxWithLabel';

//react-test-render could help to bypass the warning if component name in big char
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <CheckboxWithLabel labelOn="On" labelOff="Off" />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});