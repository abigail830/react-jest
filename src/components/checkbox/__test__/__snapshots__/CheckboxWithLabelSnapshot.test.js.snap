// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<label>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
  Off
</label>
`;

exports[`Link changes the class when hovered 2`] = `
<label>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
  Off
</label>
`;
